README.md

```diff
@@ -231,8 +231,24 @@ To edit your public profile:
     * The background and highlight colors in which your public profile should be displayed.
 
 ### Your storage
-Even if you have navigated to a different Pod, you can still view your storage.
+
+The 'Your storage' menu option brings you to an overview of all the resources part of your Pod. This will always display the storage of your WebID, so your personal storage. Even if you navigate to a different Pod, you can only still view your personal storage.
 
 To view your storage:
 1. Select **Your storage** from the dropdown menu.
-2. A Folder view of your storage is displayed.
+2. A folder view of your storage is displayed.
+
+To visualize and have access to another storage of another user you have two options at the moment:
+
+1. Add the storage directly in your profile/card.
+The store is connected to your profile with a triple like `:me space:storage </>; `. You can simply add a second triple with the same predicate `:me space:storage <otherWebID>` to your WebID (so in your file that can be found under a URI like this one: <https://timea.solidcommunity.net/profile/card#me<>). Example of triple to be added:`:me space:storage <https://suser.solidcommunity.net>:`
+
+_**remark:** make sure you have the correct prefix for space:storage, the full URI is `<http://www.w3.org/ns/pim/space#storage>`.
+_**remark:** you will only be able to visualize the resources that are publicly readable in the second Pod. Unless you have full access to the second Pod.
+
+2. Navigate to a person's Pod and login.
+You will be prompted to authorize the new person's Pod to access your Pod. This is required as a preliminary step, otherwise your Pod storage cannot be accessed and displayed. If you authorize the new WebID to access your store then you can see the store of the new Pod as a new option in the drop down header menu.
+
+In the next screenshot I (<https://timea.solidcommunity.net/profile/card#me>) was visiting the pod of <https://sharonstrats.inrupt.net/profile/card#me> and at the same time I have added <https://timea-test.solidcommunity.net/profile/card#me> as an extra store that belongs to me.
```

Comment:
```suggestion
2. Navigate to a person's Pod and log in.

   You will be prompted to authorize the new person's Pod to access your Pod. This is required as a preliminary step; otherwise, your Pod storage cannot be accessed and displayed. If you authorize the new WebID to access your store, then you can see the store of the new Pod as a new option in the drop down header menu.

   In the next screenshot, I (`<https://timea.solidcommunity.net/profile/card#me>`) was visiting the pod of `<https://sharonstrats.inrupt.net/profile/card#me>`, and at the same time, I have added <https://timea-test.solidcommunity.net/profile/card#me> as an extra store that belongs to me.
```